client/account-recovery/lost-password/index.jsx

```diff
@@ -0,0 +1,106 @@
+/**
+ * External dependencies
+ */
+import React, { Component } from 'react';
+import { findDOMNode } from 'react-dom';
+import { localize } from 'i18n-calypso';
+import classnames from 'classnames';
+
+/**
+ * Internal dependencies
+ */
+import support from 'lib/url/support';
+import Card from 'components/card';
+import Button from 'components/button';
+import FormLabel from 'components/forms/form-label';
+import FormInput from 'components/forms/form-text-input';
+
+export class LostPassword extends Component {
+	constructor() {
+		super( ...arguments );
+
+		this.state = {
+			isSubmitting: false,
+		};
+	}
+
+	onSubmit = () => {
+		const userLogin = findDOMNode( this.refs.user_login ).value;
```

Comment:
Note, that we shouldn't use the mixin, but instead setup the handlers manually: https://facebook.github.io/react/docs/two-way-binding-helpers.html#valuelink-without-linkedstatemixin

Also see: https://facebook.github.io/react/docs/forms.html#examples

@aduth or @dmsnell do you know of any newer patterns we can use here?